R/flea_playerscores.R

```diff
@@ -25,6 +25,7 @@ ff_playerscores.flea_conn <- function(conn, page_limit = NULL, ...) {
     endpoint = "FetchPlayerListing",
     sport = "NFL",
     league_id = conn$league_id,
+    season = conn$season,
```

Comment:
seems like this parameter is actually called `sort_season` https://www.fleaflicker.com/api-docs/index.html#operation--FetchPlayerListing-get - have updated for you